packages/react-events/src/dom/testing-library/domEventSequences.js

```diff
@@ -153,10 +153,11 @@ export function pointermove(target, payload) {
   }
 }
 
-export function pointerup(target, defaultPayload) {
+export function pointerup(target, defaultPayload = {}) {
   const dispatch = arg => target.dispatchEvent(arg);
   const pointerType = getPointerType(defaultPayload);
-  const payload = {buttons: buttonsType.none, ...defaultPayload};
+  // eslint-disable-next-line no-unused-vars
+  const {buttons, ...payload} = defaultPayload;
```

Comment:
This fixes a unit-test-breaking bug in the `pointerup` sequence that produced an incorrect `buttons` value for `mousedown` and produced a false-positive emulated mouse event filtering effect in the `Tap` responder.